src/components/slack/index.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react'
+import { SlackIcon } from 'components/icons'
+
+import * as S from './styles'
+
+interface Props {
+  slackLink: string
+  slackText: string
+}
+
+const Slack: React.FC<Props> = ({ slackLink, slackText }) => {
+  return (
+    <S.Link inverted to={slackLink}>
+      <>
+        <SlackIcon /> <S.ButtonLabel>{slackText}</S.ButtonLabel>
+      </>
+    </S.Link>
+  )
+}
+
+export default Slack
```

Comment:
Díky za extrakci komponenty. 👍 ... Říkám si zda, by nedávalo smysl umístit komponentu do `components/buttons`, co myslíš?

Reply:
Ahoj, určitě to tam mohu dát. 👍 Doupravím i ty zbývající části.